src/JoomlaBrowser.php

```diff
@@ -133,11 +133,12 @@ public function installJoomla()
         $I->debug('I select dk-DK as installation language');
         // Select a random language to force reloading of the lang strings after selecting English
         $I->selectOptionInChosen('#jform_language', 'Danish (DK)');
+        $I->wait(5);
         $I->waitForText('Generel konfiguration', 10, 'h3');
         // Wait for chosen to render the field
-        $I->wait(1);
         $I->debug('I select en-GB as installation language');
         $I->selectOptionInChosen('#jform_language', 'English (United Kingdom)');
+        $I->wait(5);
         $I->waitForText('Main Configuration', 10, 'h3');
```

Comment:
ok thnx, didn't know it was time waiting. will test it again, and will update the code.